modules/api-support-tiers.adoc

```diff
@@ -0,0 +1,130 @@
+
+[id="mapping-support-tiers-to-api-groups_{context}"]
+= Mapping support tiers to API groups
+
+For each support tier defined by Red Hat, we provide a mapping table for specific API groups where the upstream communities are committed to maintain forward compatibility. Any API group that does not specify an explicit compatibility level is assigned level 3 by default.
+
+[id="mapping-support-tiers-to-kubernetes-api-groups_{context}"]
+== Support for Kubernetes API groups
+
+API groups that end with the suffix `*.k8s.io `or have the form `version.<name>` with no suffix are governed by the Kubernetes deprecation policy and follow a general mapping between API version exposed and corresponding support tier unless otherwise specified.
+
+[cols="2",options="header"]
+|===
+|API version example
+|API support tier
+
+|`v1`
+|Support tier 1
+
+|`v1beta1`
+|Support tier 2
+
+|`v1alpha1`
+|Support tier 4
+
+|===
+
+[id="mapping-support-tiers-to-openshift-api-groups_{context}"]
+== Support for OpenShift API groups
+
+API groups that end with the suffix `*.openshift.io` are governed by the {product-title} deprecation policy and follow a general mapping between API version exposed and corresponding compatibility level unless otherwise specified.
+
+[cols="2",options="header"]
+|===
+|API version example
+|API support tier
+
+|`v1`
+|Support tier 1
+
+|`v1beta1`
+|Support tier 2. For example, Machine API and Autoscaling.
+
+|`v1alpha1`
+|If installed by default, Support tier 4. For example, `v1alpha1.operator.openshift.io`.
+
+|===
+
+[id="mapping-support-tiers-to-monitoring-api-groups_{context}"]
+== Support for Monitoring API groups
+
+API groups that end with the suffix `monitoring.coreos.com` have the following mapping:
+
+[cols="2",options="header"]
+|===
+|API version example
+|API support tier
+
+|`v1`
+|Support tier 1
+
+|===
+
+[id="mapping-support-tiers-to-coreos-api-groups_{context}"]
+== Support for CoreOS API groups
+
+API groups that end with the suffix `operators.coreos.com` have the following mapping:
+
+[cols="2",options="header"]
+|===
+|API version example
+|API support tier
+
+|`v1`
+|Support tier 1
+
+|`v1alpha1`
+|Support tier 3
+
+|`v1alpha2`
+|Support tier 3
+
+|===
+
+[id="mapping-support-tiers-to-metal3-api-groups_{context}"]
+== Support for Metal3 API groups
+
+API groups that end with the suffix `metal3.io` have the following mapping:
+
+[cols="2",options="header"]
+|===
+|API version example
+|API support tier
+
+|`v1alpha1`
+|Support tier 2. For example, `BareMetalHost` and `Provisioning`.
```

Comment:
Someday, these will graduate to beta and then GA, so it might be easier to omit these so we don't ever have to update this file for API version changes.